tsconfig.json

```diff
@@ -1,96 +1,17 @@
 {
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
Comments are not valid JSON and as such caused the `ts-mocha` dependency to choke.